dashboard/test/factories/pd_factories.rb

```diff
@@ -45,7 +45,7 @@
         workshop.enrollments << build(:pd_enrollment, workshop: workshop)
       end
       evaluator.enrolled_and_attending_users.times do
-        teacher = create :teacher
+        teacher = create :teacher, email: "teacher#{SecureRandom.hex}@code.org"
```

Comment:
why do we need this?

Reply:
I added this because teacher creation failed on my local machine. This is (I think) because the FactoryGirl sequence gets reset between test runs, so every time I run it, this line would try to create with the sequence used by the email at 1.

I'm iffy on keeping this. I want people to run tests locally and this will enable it.

Reply:
I agree it's hacky, and I don't like this solution:
 1. If creating teachers is a problem, we should update the underlying `teacher` factory, not this specific call. 
 2. We still can have conflicts with a random hex, just unlikely. Can we come up with a better solution, maybe something more like what we do in the [school factory](https://github.com/code-dot-org/code-dot-org/blob/fde761a03b882b8626510467af798df55db3e650/dashboard/test/factories/factories.rb#L734-L734)?

Reply:
Btw, it looks like this is a known issue in FactoryBot, that doesn't get much attention: https://github.com/thoughtbot/factory_bot/issues/615